src/test/java/uk/gov/companieshouse/mapper/email/DissolutionEmailMapperTest.java

```diff
@@ -89,6 +90,24 @@ public void mapToApplicationRejectedEmailData_mapsApplicationRejectedEmailData()
         assertEquals(applicationRejectedEmailData.getRejectReasons(), result.getRejectReasons());
     }
 
+    @Test
+    public void mapToPendingPaymentEmailData() {
+        final Dissolution dissolution = generateDissolution();
+        final PendingPaymentEmailData pendingPaymentEmailData = EmailFixtures.generatePendingPaymentEmailData();
+
+        when(environmentConfig.getChsUrl()).thenReturn(CHS_URL);
+
+        final PendingPaymentEmailData result = dissolutionEmailMapper.mapToPendingPaymentEmailData(dissolution);
+
+        assertEquals(pendingPaymentEmailData.getTo(), result.getTo());
+        assertEquals(pendingPaymentEmailData.getSubject(), result.getSubject());
+        assertEquals(pendingPaymentEmailData.getDissolutionReferenceNumber(), result.getDissolutionReferenceNumber());
+        assertEquals(pendingPaymentEmailData.getCompanyNumber(), result.getCompanyNumber());
+        assertEquals(pendingPaymentEmailData.getCompanyName(), result.getCompanyName());
+        assertEquals(pendingPaymentEmailData.getChsUrl(), result.getChsUrl());
+        assertEquals(pendingPaymentEmailData.getCdnHost(), result.getCdnHost());
```

Comment:
I don;t like these assertions. You are basically copying the implementation of the mapper. You are also relying on the fixtures containing valid data. This unit test would continue to pass if the fixtures was full of null values. Instead follow the example set by the mapToSignatoryToSignEmailData tests and hardcode the data you are asserting on.